client/cmdhfmfp.c

```diff
@@ -439,7 +448,7 @@ int CmdHFMFPRdbl(const char *cmd) {
 	bool verbose = arg_get_lit(1);
 	int blocksCount = arg_get_int_def(2, 1);
 	bool keyB = arg_get_lit(3);
-	int plain = arg_get_lit(4) | true;
+	int plain = arg_get_lit(4);
```

Comment:
Now that we have a "Plain" flag, the help text should be without the "in plain mode"